### ISSUE TITLE
[App] Add Image Hub as a sample app

### ISSUE THREAD

**Issue by leecalcote** [layer5io/meshery-consul#58]
**Enhancement Description**
A new sample app is available as a workload on Consul. The name of this app is Image Hub - https://github.com/layer5io/image-hub. This app should be available through the Consul adapter.

**Comment by kanishkarj** [comment]
I'll take this up.

**Comment by kanishkarj** [comment]
I think something we'll need here before implementing this is for consul install to accept a helm config. We might want to do this with other adapters too. 

Any thoughts on this @lee?

**Comment by leecalcote** [comment]
@kanishkarj Yes, you’re right. Helm and as a universal capability. 

@Aisuko do you have an update on progress here?  // @nveenjain 

@kanishkarj for now, assume that helm or other is already in place.

**Comment by Aisuko** [comment]
Yep, we already have a project for using Helm to render the chart to Kubernetes manifest(Includes  Linkerd and Consul). In my case, I have another though is may we use Helm client to deploy the chart? Because we use client-go create resource one-by-one may not good for debugging what's going on

**Comment by kushthedude** [comment]
@kanishkarj Is this issue addressed ?

**Comment by kanishkarj** [comment]
Kind of, the pr referencing this has been closed. But there is still one issue to address, that is how deployment of consul with custom helm config. @Aisuko might throw more light on this.

**Comment by kanishkarj** [comment]
@Aisuko, I personally do feel we should use helm client to deploy consul, and any service mesh that supports helm. It makes maintanence easier. @leecalcote  thoughts on this?

**Comment by kushthedude** [comment]
@kanishkarj https://github.com/layer5io/meshery-consul/issues/37 is this the issue that is left to be addressed ?

**Comment by kanishkarj** [comment]
Yes, that's the exact issue.

**Comment by Aisuko** [comment]
> @Aisuko, I personally do feel we should use helm client to deploy consul, and any service mesh that supports helm. It makes maintanence easier. @leecalcote thoughts on this?

For sure, we should do this with Helm v3

**Comment by leecalcote** [comment]
@Aisuko yes, I agree. While there are other possible approaches (i.e. Shipyard) under consideration in the future, ‘helm’ is what we have here and now. Yes, please proceed.
